Project-iOS/XLProjectName/XLProjectName/Networking/Manager.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  Manager.swift
+//  XLProjectName
+//
+//  Created by XLAuthorName ( XLAuthorWebsite )
+//  Copyright © 2016 XLOrganizationName. All rights reserved.
+//
+
+import Foundation
+import Opera
+import Alamofire
+import KeychainAccess
+
+class NetworkManager: RxManager {
```

Comment:
Rx manager should update authentication token if needed. At least we should provide the commented code.